### PR TITLE
feat(content): created accessibility-statement page

### PR DESCRIPTION
**Pull request by alittlecross**
- added link text to app locale
- added link to layout helper
- removed unnecessary footer-links file
- added route to general routes
- added test for function
- added function to general functions
- added namespace to i18n config
- created locale
- created page

Jira SA 2947

Signed-off-by: Paul Fazackerley <paul.fazackerley@engineering.digital.dwp.gov.uk>

### CODE REVIEW

File: locales/en/accessibility-statement.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "h2-6": "What to do if you cannot access parts of this service",
+  "p14": "If you need State Pension information in a different format, for example, large print, Braille or audio recording, phone:",
+  "p14-li1": "0800 731 7898 (UK)",
+  "p14-li2": "+ 44 (0)191 218 2051 (textphone)",
+  "p14-li3": "+44 (0)191 2183600 (outside the UK)",
+  "p15": "Other Pension Service contact details are available online, at <a href='https://www.gov.uk/contact-pension-service' class='govuk-link'>https://www.gov.uk/contact-pension-service</a>",
+  "h2-7": "Reporting accessibility problems with this service",
+  "p16": "We’re always looking to improve the accessibility of this service.",
+  "p17": "Contact us if you find any problems not listed on this page, or if you think we’re not meeting accessibility requirements.",
+  "p18": "Email details about the issue to <a href=mailto:'__EMAIL__' class='govuk-link'>statepension.accessibility@dwp.gov.uk</a>",
```
Comment (malcolmhire) [inline locales/en/accessibility-statement.json:40]: Same as welsh

File: locales/cy/accessibility-statement.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  "h2-6": "Beth i’w wneud os na allwch fynychu rannau’r gwasanaeth hwn",
+  "p14": "Os bydd angen gwybodaeth Pensiwn y Wladwriaeth mewn fformat gwahanol, er enghraifft print bras, Braille neu sain, ffoniwch:",
+  "p14-li1": "0800 731 7898 (DU)",
+  "p14-li2": "+ 44 (0)191 218 2051 (ffôn testun)",
+  "p14-li3": "+44 (0)191 2183600 (o tu allan y DU)",
+  "p15": "Mae manylion cyswllt eraill y Gwasanaeth Pensiwn ar gael ar-lein, yn <a href='https://www.gov.uk/cysylltwch-gwasanaeth-pensiwn' class='govuk-link'>https://www.gov.uk/cysylltwch-gwasanaeth-pensiwn</a>",
+  "h2-7": "Rhoi gwybod am broblemau hygyrchedd gyda’r gwasanaeth hwn",
+  "p16": "Rydym bob amser yn ceisio gwella hygyrchedd y gwasanaeth hwn.",
+  "p17": "Cysylltwch ȃ ni os byddwch yn dod o hyd i unrhyw broblemau nad ydynt wedi’u rhestru ar dudalen hon, neu os ydych yn meddwl nad ydym yn cwrdd ȃ gofynion hygyrchedd.",
+  "p18": "Anfonwch ebost gyda manylion y broblem i <a href=mailto:'__EMAIL__' class='govuk-link'>statepension.accessibility@dwp.gov.uk</a>",
```
[the remark this answers is not in the document]
Comment (malcolmhire) [reply locales/cy/accessibility-statement.json:40]: Maybe do this instead:
```suggestion
  "p18": "Anfonwch ebost gyda manylion y broblem i <a href='mailto:__EMAIL__' class='govuk-link'>__EMAIL__</a>",
```

File: locales/en/app.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
   "navigation": {
      "general": "to",
      "auth": "Your details",
+     "accessibilityStatement": "Accessibility Statement",
```
Comment (malcolmhire) [inline locales/en/app.json:25]: From content design, needs to be lowercase statement:```suggestion
     "accessibilityStatement": "Accessibility statement",
```

File: locales/cy/accessibility-statement.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (alittlecross) [reply locales/cy/accessibility-statement.json:40]: incorporated suggestion

File: locales/en/app.json (hunk 3 above)
Comment (alittlecross) [reply locales/en/app.json:25]: incorporated suggestion

File: locales/en/accessibility-statement.json (hunk 1 above)
Comment (alittlecross) [reply locales/en/accessibility-statement.json:40]: incorporated suggestion

**Comment by malcolmhire** [comment]
Pull request merged in GitLab

**Pull request closed without merging**